pkg/config/config.go

```diff
@@ -612,3 +616,17 @@ func IsKubernetes() bool {
 	}
 	return false
 }
+
+// SetOverrides provides an externally accessible method for
+// overriding config variables. Used by Android to set
+// the various config options from intent extras
+func SetOverrides(vars map[string]interface{}) {
```

Comment:
this has to be called before `config.Load` to take effect, let's reflect this in the function's docs